modules/users/manifests/alexsmith.pp

```diff
@@ -0,0 +1,8 @@
+# Create the alexsmith user
+class users::alexsmith {
+  govuk_user { 'alexsmith':
+    fullname => 'Alex Smith',
+    email    => 'alex.smith@digital.cabinet-office.gov.uk',
```

Comment:
I think the email address in the user reviewer is only used in the AWS checks.  So if your AWS account is `alex.smith@...`, we'll have to update the reviewer.  If your AWS account is `alex.whitehead-smith@...`, I think it'd be better to use the same email address here for consistency.

Reply:
Cool, that makes sense. I only just put in a request to create a `gds-users` AWS account (the DMp managed their own accounts), so I'll put in a new one that uses `alex.smith@...` and it should all be ok 🤞